Troubleshooting.md

```diff
@@ -126,15 +126,28 @@ We should manually skip the bundle in `recieving.presence_absence` and wait for
         and document ->> 'collection' is not null
     )
 
-    select
-        *
-    from error_records
-    join complete_records using (sample)
+    select *
+    from warehouse.sample
+    where sample_id in
+    (
+      select distinct(cast(b.plog ->> 'sample_id' as int)) as error_sample_id
+      from
+      (
+        select jsonb_array_elements(a.processing_log) as plog
+        from
+        (
+          select err.processing_log
+          from error_records err
+          join complete_records comp using (sample)
+        ) as a
+      ) as b
+    where b.plog @> '{"etl":"manifest"}'
+    order by cast(b.plog ->> 'sample_id' as int)
```

Comment:
FWIW, this is another way to write this subquery: 
```sql
   select
        p."sample_id"
    from error_records
    join complete_records using (sample),
    jsonb_to_recordset(error_records.processing_log) as p("etl" text, "sample_id" int)
    where p."etl" = 'manifest'
    order by p."sample_id"
```

Reply:
Nice. Lots of ways to do the same thing.